site/content/resources/entrepreneurship/shoplatinx.md

```diff
@@ -0,0 +1,9 @@
+---
+title: ShopLatinx
+image: "/assets/img/resources/shoplatinx.png"
+description: Online directory of Latinx-owned businesses and editorial content fostering Latinx entrepreneurship.
+categories:
```

Comment:
We don't usually use categories but we probably will in the future so this is great!